xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -235,9 +240,19 @@ in more data asynchronously before continuing with a new instance of the <xref:S
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Parses the current JSON token value from the source and decodes the base 64 encoded JSON string as a byte array.</summary>
+        <returns>The byte array that represents the current JSON token value.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.InvalidOperationException">The type of the JSON token is not a <see cref="F:System.Text.Json.JsonTokenType.String" />.</exception>
+        <exception cref="T:System.FormatException">The JSON string contains data outside of the expected base 64 range.
```

Comment:
Should we also add the similar level of detail to the `JsonElement.GetBytesFromBase64` API?

Reply:
I'm not sure what you mean. Can you please elaborate? Or was the question directed at the other Json owners?

Reply:
The `JsonElement.GetBytesFromBase64` exception documentation doesn't show all the cases where it could fail (i.e. where the data would be invalid and cannot be decoded as Base64 string). The `Utf8JsonReader.GetBytesFromBase64` provides more details.

I am asking if we should add the same info to  `JsonElement.GetBytesFromBase64`, which just says:
`The value is not encoded as base 64 text and hence cannot be decoded to bytes.`
https://github.com/dotnet/dotnet-api-docs/blob/771baa41a5149ce706b3c99d6cd698683a5616fe/xml/System.Text.Json/JsonElement.xml#L187

I say, yes.
@bartonjs?

Reply:
I think Utf8JsonReader is overly descriptive.  They all just mean "The JSON string is not a valid base 64-encoded payload." which is what I would use there, and here.

Reply:
The info about what is considered "invalid" is useful, imo, but I can see it not being relevant to JSON (since its primarily about Base64). I would prefer to keep this detail as I don't see a down-side for having it, but don't feel too strongly (since the Base64 API already sufficiently document what is considered "invalid" - https://docs.microsoft.com/en-us/dotnet/api/system.buffers.text.base64.decodefromutf8?view=netstandard-2.1#remarks).

Reply:
Should we do something about this or not? Can you provide a code suggestion if yes? @bartonjs @ahsonkhan

Reply:
Chatted with Ahson and decided to substitute the long exception message with the shorter one. Fixed.